.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.7.3
+  hooks:
+    # Run the linter.
+    - id: ruff
+      args: [ --fix ]
+    # Run the formatter.
+    - id: ruff-format
+repos:
```

Comment:
Pretty sure ruff can sort imports.

Reply:
Alright, updated.

Reply:
It can, but so far, I don't like what it's doing when we trigger it with pre-commit.  The "way" is to run a filtered check, and it doesn't seem confined to the diff. I'm going to revert these last few commits & stick with isort for now.

Reply:
Created #569 to address this in the future.